addOns/alertFilters/src/main/java/org/zaproxy/zap/extension/alertFilters/AlertFilterAPI.java

```diff
@@ -77,7 +82,15 @@ public AlertFilterAPI(ExtensionAlertFilters extension) {
                         ACTION_ADD_ALERT_FILTER,
                         new String[] {PARAM_CONTEXT_ID, PARAM_RULE_ID, PARAM_NEW_LEVEL},
                         new String[] {
-                            PARAM_URL, PARAM_URL_IS_REGEX, PARAM_PARAMETER, PARAM_ENABLED
+                            PARAM_URL,
+                            PARAM_URL_IS_REGEX,
+                            PARAM_PARAMETER,
+                            PARAM_PARAMETER_IS_REGEX,
+                            PARAM_ATTACK,
+                            PARAM_ATTACK_IS_REGEX,
+                            PARAM_EVIDENCE,
+                            PARAM_EVIDENCE_IS_REGEX,
+                            PARAM_ENABLED,
```

Comment:
This will create "problems" when generating the clients, the `enabled` would have to be kept in the same place.